### PR TITLE
html: build html page in rust with maud

### PR DESCRIPTION
**Pull request by lvaccaro**
Remove index.html and build html page from rust with maud library. 
Follow @RCasatta code of his project: https://github.com/RCasatta/rusty-paper-wallet

### CODE REVIEW

File: src/html.rs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                meta http-equiv="Refresh" content=(meta_http_content);
+                title { (address) }
+                link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous" {}
+                link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.4.0/font/bootstrap-icons.css" {}
+                style { (CSS) }
+            }
+            body {
+                (inner_header())
+                main role="main" class="container" {
+                    (inner_network("Bitcoin"))
```
Comment (notmandatory) [inline src/html.rs:98]: Should the network get passed in as a function argument and used here? ie. "Bitcoin", "Testnet", "Regtest", "Signet"?

File: src/html.rs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                i.bi.bi-github role="img" aria-label="GitHub" {}
+            }
+        }
+    };
+    header
+}
+
+fn inner_network(network: &str) -> Markup {
+    let partial = html! {
+        div class="d-flex align-items-center p-3 my-3 text-white-50 bg-orange rounded box-shadow" {
```
Comment (notmandatory) [inline src/html.rs:21]: Would be nice to use a different background color for non-main networks, some use blue for testnet/regtest.

**Review by notmandatory — COMMENTED** [review]
Great idea. I did a quick review and local test and LGTM. 

I added a couple comments but they could also be addressed in future PRs.

File: src/html.rs (hunk 2 above)
Comment (lvaccaro) [reply src/html.rs]: thanks for the suggestion... add color class parameter based on network type

File: src/html.rs (hunk 1 above)
Comment (lvaccaro) [reply src/html.rs]: thanks, passing network from conf file to html builder

**Comment by notmandatory** [comment]
Looks good! I tested with each network and verified color and text.

**Pull request merged**